config/unix-xl.cmake

```diff
@@ -26,16 +25,16 @@ if( NOT CXX_FLAGS_INITIALIZED )
   # On Darwin, we also need this config file:
   # -F/projects/opt/ppc64le/ibm/xlc-16.1.1.2/xlC/16.1.1/etc/xlc.cfg.rhel.7.5.gcc.7.3.0.cuda.9.2
   # -qfloat=nomaf -qxlcompatmacros
-  set( CMAKE_C_FLAGS                "-g" ) # -qarch=auto
+  set( CMAKE_C_FLAGS                "-g --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1" ) # -qarch=auto
```

Comment:
This is required for rzansel/sierra, but will probably break other XL-based builds (e.g. Darwin).  We will deal with that separately.